assets/app/view/spreadsheet.rb

```diff
@@ -79,14 +86,14 @@ def render_title
           h(:th, { attrs: { colspan: or_history_titles.size } }, 'OR History'),
           ]),
         h(:tr, [
-          h(:th, { style: { width: '20px' } }, ''),
+          render_sort_link({ style: { width: '20px' } }, 'SYM', 'ID'),
           *@game.players.map { |p| h(:th, props, p.name) },
           h(:th, props, 'IPO'),
           h(:th, props, 'Market'),
           h(:th, props, 'IPO'),
-          h(:th, props, 'Market'),
-          h(:th, props, 'Cash'),
-          h(:th, props, 'Operating Order'),
+          render_sort_link(props, 'Market', 'SHARE-PRICE'),
```

Comment:
let's call it SHARE_PRICE and OPERATING_ORDER

Reply:
sure